<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSelectionModelTestWidget</class>
 <widget class="QWidget" name="ItemSelectionModelTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item selection model sandbox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbResetModel">
       <property name="text">
        <string>Reset model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDisplayCurrentIndex">
       <property name="text">
        <string>Display current index</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRemove">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPrepend">
       <property name="text">
        <string>Prepend</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAppend">
       <property name="text">
        <string>Append</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="useSortFilterProxyModel">
       <property name="text">
        <string>use sort filter proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterCriteria">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QListView" name="listView"/>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="tableViewCurrentIndexLabel">
         <property name="text">
          <string>Current index:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="tableViewCurrentIndex">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="tableViewCurrentChangedEmittedLabel">
         <property name="text">
          <string>Emitted:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="tableViewCurrentChangedEmitted">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QTreeView" name="treeView"/>
     </item>
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="listViewCurrentIndexLabel">
         <property name="text">
          <string>Current index:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="listViewCurrentIndex">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="treeViewCurrentIndexLabel">
         <property name="text">
          <string>Current index:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="treeViewCurrentIndex">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="0" column="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Selection model</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbQtSelectionModel">
          <property name="text">
           <string>Qt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMdtSelectionModel">
          <property name="text">
           <string>Mdt</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
